### PR TITLE
Fix security token filter chain

### PR DESCRIPTION
**Pull request by panel14**

### CODE REVIEW

File: src/main/java/org/itmo/eventapp/main/security/config/SecurityConfig.java [hunk 1]
```diff
@@ -15,47 +16,55 @@
...
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 
-@EnableMethodSecurity
-@EnableWebSecurity
+import java.util.Arrays;
+
+
+//@EnableWebSecurity
```
Comment (IP-13) [inline src/main/java/org/itmo/eventapp/main/security/config/SecurityConfig.java:28]: а это не надо больше?

**Review by IP-13 — APPROVED** [review]

File: src/main/java/org/itmo/eventapp/main/security/config/SecurityConfig.java (hunk 1 above)
Comment (panel14) [reply src/main/java/org/itmo/eventapp/main/security/config/SecurityConfig.java:29]: да, можно не указывать

**Review by omgronny — APPROVED** [review]

**Comment by IP-13** [comment]
|Overall Project|63.98% **`-2.46%`**|:green_apple:|
|:-|:-|:-:|
|Files changed|32.28%|:x:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[SecurityConfig.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Fconfig%2FSecurityConfig.java)|100%|:green_apple:|
|[JwtTokenUtil.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Futil%2FJwtTokenUtil.java)|98.78%|:green_apple:|
|[MiscSecurityExpression.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Fsecurityexpression%2FMiscSecurityExpression.java)|98.5%|:green_apple:|
|[CommonControllerAdvice.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fexceptionhandling%2FCommonControllerAdvice.java)|78.18% **`-10.91%`**|:x:|
|[JwtFilter.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Ffilter%2FJwtFilter.java)|78.18% **`-21.82%`**|:x:|
|[AuthenticationService.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FAuthenticationService.java)|22.44% **`-2.76%`**|:x:|
|[ExceptionResponse.java](https://github.com/itmo-events-app/event-app/blob/92cae3f4157f3325c1b1ecbaff412b5c69ac5538/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2FexceptionDto%2FExceptionResponse.java)|0%|:x:|

**Comment by IP-13** [comment]
|Overall Project|62.74% **`-2.4%`**|:green_apple:|
|:-|:-|:-:|
|Files changed|32.28%|:x:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[SecurityConfig.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Fconfig%2FSecurityConfig.java)|100%|:green_apple:|
|[JwtTokenUtil.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Futil%2FJwtTokenUtil.java)|98.78%|:green_apple:|
|[MiscSecurityExpression.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Fsecurityexpression%2FMiscSecurityExpression.java)|98.5%|:green_apple:|
|[CommonControllerAdvice.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fexceptionhandling%2FCommonControllerAdvice.java)|78.18% **`-10.91%`**|:x:|
|[JwtFilter.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Ffilter%2FJwtFilter.java)|78.18% **`-21.82%`**|:x:|
|[AuthenticationService.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FAuthenticationService.java)|17.75% **`-2.07%`**|:x:|
|[ExceptionResponse.java](https://github.com/itmo-events-app/event-app/blob/76fd14409f861ac2e74a3c79eb48e42ba430deaf/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2FexceptionDto%2FExceptionResponse.java)|0%|:x:|

**Pull request merged**